scripts/release/prepare-canary-commands/get-latest-master-build-number.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const http = require('request-promise-json');
+const {logPromise} = require('../utils');
+
+const run = async () => {
+  // https://circleci.com/docs/api/#recent-builds-for-a-project-branch
+  const metadataURL = `https://circleci.com/api/v1.1/project/github/facebook/react/tree/master`;
+  const metadata = await http.get(metadataURL, true);
+  const build = metadata.find(entry => entry.branch === 'master').build_num;
```

Comment:
Do you want the latest completed+successful build instead of just the latest? Failed and incomplete both seem subpar.

Reply:
🤔 Good point.